apps/occurrences/serializers.py

```diff
@@ -9,8 +9,8 @@ class OccurrenceSerializer(SearchableModelSerializer):
     """Serializer for occurrences."""
 
     date_html = serpy.Field()
-    summary = serpy.MethodField()
-    description = serpy.MethodField()
+    summary = serpy.Field(attr='summary.html')
```

Comment:
This should be good, since `summary` and `description` are required fields on Occurrence.

Reply:
Offtopic but how does `attr='summary.html'` exactly work? I [read](https://serpy.readthedocs.io/en/latest/api.html#serpy.Field) that `attr` is the attribute to get on the object but I couldn't understand where `summary.html` is processed (I'm guessing adding `.html` to field's must be doing HTML escaping or similar)

Reply:
Yep, we have a custom HTMLField implementation (see https://github.com/ModularHistory/modularhistory/blob/4a187f44f12f33ab543a8fc2cb91d3236f5217d5/modularhistory/fields/html_field.py#L133). `summary` is an HTMLField on the Occurrence model, so `summary.html` returns the processed/escaped HTML for that field.